.buildkite/common-pipeline.go

```diff
@@ -0,0 +1,19 @@
+package buildkite
+
+import "fmt"
+
+func TrivyStepGenerator(trivyPlugin, shellPlugin string) {
```

Comment:
What do you think about having this function take an `io.Writer` (for example: `os.Stdout`) and returning an error?

This would allow users to specify where they want the template to be written to.

For example, the new function signature would look like this:

```
func TrivyStepGenerator(trivyPlugin, shellPlugin string, w io.Writer) error
```

Reply:
Thanks for suggestion I've addressed it.